app/Http/Controllers/Auth/LoginController.php

```diff
@@ -67,20 +68,30 @@ protected function verifyLogin(Request $request) {
 
     protected function vatusaAuth($resourceOwner, $accessToken) {
         $client = new Client();
+
+        if (App::environment('local')) {
+            $resourceOwner->data->cid = "10000002";
+            $resourceOwner->data->personal->email = "dev@ztl.local";
+        }
+
         $result = $client->request('GET', Config::get('vatusa.base').'/v2/user/' . $resourceOwner->data->cid . '?apikey=' . Config::get('vatusa.api_key'), ['http_errors' => false]);
         $realops_toggle_enabled = toggleEnabled('realops');
 
         if (! $result || $result->getStatusCode() != 200) {
             if ($realops_toggle_enabled) {
```

Comment:
Why not simply this as follows:
`if ($realops_toggle_enabled && !App::environment('local')) {`